src/scene-entry-manager.js

```diff
@@ -202,11 +202,23 @@ export default class SceneEntryManager {
   };
 
   _pinElement = el => {
-    const networkId = el.components.networked.data.networkId;
+    const { networkId } = el.components.networked.data;
+
+    const { fileId, src } = el.components["media-loader"].data;
+
+    let fileToken, promotionToken;
```

Comment:
I think the term we've been using is `access_token`, perhaps we should update the query string arg as well